src/sections/dataset/dataset-versions/DatasetVersions.tsx

```diff
@@ -111,36 +111,32 @@ export function DatasetVersions({
           </thead>
           <tbody>
             {datasetVersionSummaries?.map((dataset, index) => {
-              const previousVersion =
-                index < datasetVersionSummaries.length - 1
-                  ? datasetVersionSummaries[index + 1]
-                  : null
+              const isVersionDeaccessioned = (version: DatasetVersionSummaryInfo) =>
```

Comment:
This logic for defining isDeaccessioned is also used in defining selectableVersions, can it be replaced with a function that is called in both places?